tests/query/highlights/fusion/basic.fusion

```diff
@@ -0,0 +1,61 @@
+include: SomeFile.fusion
+//<- include
+//     ^punctation.delimiter
+//       ^uri
```

Comment:
is this really an URI? It's kind of. I'm just not very sure about that.

Reply:
also typo here: ^punctation.delimiter

Reply:
hmm - i would say so, at least it looks like the best fit of the given highlight groups

includes in fusion can be one of three forms
1. files relative to the current file (the given case)
2. globbing patterns relative to the current file `**/*.fusion`
3. full specification with a resource schema lik `resource://SomePackage/Resources/**/*.fusion`, that also could be a globbing

Reply:
Then it's fine!